doc/env_vars.md

```diff
@@ -104,6 +104,8 @@ The following options are used for advanced database performance tuning and may
 | `LRSQL_OIDC_SCOPE_PREFIX`       | `oidcScopePrefix`       | An optional prefix prepended to OIDC scope. For example, setting this to `lrs:` would change the expected `all` scope to `lrs:all`                                                                                | `""`                                                                                  |
 | `LRSQL_STMT_RETRY_LIMIT`        | `stmtRetryLimit`        | The number of times to retry a statement post transaction before failing.                                                                                                                                         | `10`                                                                                  |
 | `LRSQL_STMT_RETRY_BUDGET`       | `stmtRetryBudget`       | The max amount of time allowed for statement POST transaction retries before failing (ms).                                                                                                                        | `1000`                                                                                |
+| `LRSQL_ENABLE_REACTIONS`        | `enableReactions`       | Whether or not to enable statement reactions.                                                                                                                                                                     | `false`                                                                               |
+| `LRSQL_REACTION_BUFFER_SIZE`    | `reactionBufferSize`    | Number of pending reactions to allow.                                                                                                                                                                             | `10000`                                                                               |
```

Comment:
Add some comment here about what happens to overflow.

Reply:
Will do. Also gonna make a separate doc page on reactions outside this pr.

Reply:
https://github.com/yetanalytics/lrsql/pull/316/commits/689f9540c19f4d0ca4f77a51e2baa3cd746a3d19